CIP-1854/CIP-1854.md

```diff
@@ -0,0 +1,245 @@
+---
+CIP: 1854
+Title: Multi-signatures HD Wallets
+Authors: Matthias Benkort <matthias.benkort@iohk.io>, Pawel Jakubas <pawel.jakubas@iohk.io>
+Comments-Summary: Multi-party transaction signing and key management for HD wallets.
+Comments-URI: https://github.com/cardano-foundation/CIPs/wiki/Comments:CIP-1854
+Status: Draft
+Type: Standards
+Created: 2021-02-23
+License: CC-BY-4.0
+---
+
+# Abstract
+
+This document describes how to realize multi-parties transactions in Cardano. It is defined as an alternative to [CIP-1852] for Cardano HD wallets. This specification does not cover the transport and sharing of partially signed transactions which is / will be covered in another document.
+
+## Glossary
+
+
+Term     | Definition
+---      | ---
+HD       | Hierarchical Deterministic, refers to wallets as described in [BIP-0032].
+Multisig | Shorthand for Multi-party signature scheme. 
+
+# Motivation
+
+## Overview 
+
+Multisig wallets are Cardano wallets capable of providing and tracking keys involved in multi-parties transactions. Such transactions are used to move funds from addresses where (typically) more than one user must sign transactions for its funds to be spent. A simple analogue is a joint bank account where two account holders must approve withdrawals from the account. Cardano has native support for multi-signature schemes: funds at a multi-signature script address are owned by a monetary script, which specifies one or more combinations of cryptographic signatures which must be present to unlock funds at the address (see the [Shelley Ledger Spec, figure 5][delegation_design.pdf]. In a similar fashion, Cardano multisig scripts can also be used to capture stake rights on a particular address, shared between different parties. 
```

Comment:
There is no figure 5 in https://hydra.iohk.io/build/4425240/download/1/delegation_design_spec.pdf.

Reply:
Whoops. That's the wrong document being linked here. It's supposed to be the formal specification, and it's now figure 4 apparently :| 

![Screenshot from 2021-04-28 13-19-07](https://user-images.githubusercontent.com/5680256/116395382-58f9ff80-a824-11eb-92e5-1bc3f3272b2e.png)

Reply:
Good catch @janmazak, thanks! Fixed in: https://github.com/cardano-foundation/CIPs/pull/69/commits/0b02d763629d62923d7b8a3d24dd12f919a4f446